gcc/rust/rust-session-manager.cc

```diff
@@ -53,7 +54,65 @@ const char *kHIRDumpFile = "gccrs.hir.dump";
 const char *kHIRTypeResolutionDumpFile = "gccrs.type-resolution.dump";
 const char *kTargetOptionsDumpFile = "gccrs.target-options.dump";
 
-const std::string kDefaultCrateName = "example";
+const std::string kDefaultCrateName = "rust_out";
```

Comment:
is this the default crate name in Rustc? @flip1995 ?

Reply:
[Yes](https://github.com/rust-lang/rust/blob/2a83fbc42a9bb6bfdb8d3fb4ecce83fb410d7642/compiler/rustc_session/src/output.rs#L91) if no `--crate-name` was given AND no `#![crate_name]` attr was found AND there is no filename.

Reply:
Ah ok let's raise an issue for this then, I used this to default to example so that on compiler explorer we can compare the outputs and assumed the default crate name was an example for rustc.